script.js

```diff
@@ -257,4 +257,27 @@ contactForm.addEventListener('submit', (e) => {
     validationMsg.textContent ='';
     contactForm.submit();
   }
-});
\ No newline at end of file
+});
+
+// Local storage
+if (localStorage.getItem('formData')) {
+  const formData = JSON.parse(localStorage.getItem('formData'));
+  emailInput.value = formData.emailAddress;
+  nameInput.value = formData.fullName;
+}
+
+contactForm.addEventListener('input', () => {
+  const formData = {
+    fullName: nameInput.value,
+    emailAddress: emailInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
+});
+
+contactForm.addEventListener('input', () => {
+  const formData = {
+    fullName: nameInput.value,
+    emailAddress: emailInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
```

Comment:
- [ ] Well done implementing the user input persistence on page reload. However, I notice that you omitted the textarea-body input. Kindly ensure it also saves data to the local storage. 
```javascript
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```